guides/source/action_cable_overview.md

```diff
@@ -626,11 +626,11 @@ key being passed to `current_user.appear`.
 
 The appearance example was all about exposing server functionality to
 client-side invocation over the WebSocket connection. But the great thing
-about WebSockets is that it's a two-way street. So now let's show an example
+about WebSockets is that it's a two-way street. So now, let's show an example
```

Comment:
I think this reads fine without the comma, but, if we do add that comma, we should add another:

```suggestion
about WebSockets is that it's a two-way street. So, now, let's show an example
```